### PR TITLE
Add log collector flags to README

### PR DESCRIPTION
**Pull request by pgombar**
## Description

Update README with new feature flags.

---

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and Travis.CI is passing.

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/WALinuxAgent/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by larohra — APPROVED** [review]

**Review by larohra — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -180,6 +180,8 @@ For CoreOS, use:
 
 `-start`: Run waagent as a background process
 
+`-collect-logs [-full]`: Runs the log collector utility that produces a zip with the relevant log files useful for support. Use flag `-full` for more exhaustive log collection.
```
Comment (narrieta) [inline README.md:183]: let's add a little info as to what the tool does. i'd just mention it collects logs for debugging and where it places the zip

Comment (pgombar) [reply README.md:183]: "Runs the log collector utility that collects relevant agent logs for debugging and stores them in the agent folder on disk. Exact location will be shown when run. Use flag `-full` for more exhaustive log collection."
What do you think of this?

Comment (narrieta) [reply README.md:183]: ok

**Review by narrieta — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -180,6 +180,8 @@ For CoreOS, use:
 
 `-start`: Run waagent as a background process
 
+`-collect-logs [-full]`: Runs the log collector utility that collects relevant agent logs for debugging and stores them in the agent folder on disk. Exact location will be shown when run. Use flag `-full` for more exhaustive log collection.
```
Comment (larohra) [inline README.md:183]: >stores them in the agent folder on disk

Would it be better to specify the exact directory here?

**Review by larohra — APPROVED** [review]
1 minor comment else LGTM

File: README.md (hunk 2 above)
Comment (pgombar) [reply README.md:183]: It's configurable.

**Pull request merged**